client/src/components/TrackGroup/TrackGroupEmbed.tsx

```diff
@@ -87,18 +88,20 @@ const TrackGroupEmbed: React.FC<{
           </code>
         </div>
       </Modal>
-      <li
-        className={css`
-          margin-top: 0rem;
-          font-size: 1.2rem;
-          background: transparent;
-          color: var(--mi-primary-color);
-          margin-left: 0.2rem;
-        `}
-        onClick={() => setIsPopupOpen(true)}
-      >
-        <Button>{t("share")}</Button>
-      </li>
+      <div onClick={() => setIsPopupOpen(true)}>
```

Comment:
hmm, is there a reason the onClick isn't on the Button?

Reply:
Ok done !